tapir/log/views.py

```diff
@@ -126,6 +127,14 @@ def __init__(self, *args, **kwargs):
     time = DateRangeFilter(field_name="created_date")
     actor = TapirUserModelChoiceFilter(label=_("Actor"))
     members = ShareOwnerModelChoiceFilter(method="member_filter", label=_("Member"))
+    log_class_type__model = django_filters.ChoiceFilter(
+        choices=[
+            (item.model, item)
```

Comment:
I think we need custom names : even if we filter out the "app |" part the and "log entry" part, "update shift user data" or "text" won't make sense for a normal user. Custom names would also enable translations. 
Something like this should work : 
`(item.model, item.model_class().verbose_name if hasattr(item.model_class(), "verbose_name") else item)`
Then we add a `verbose_name = _("A test name")` to the log classes.
There's maybe a better name than `verbose_name`. Maybe also an abstract method in `LogEntry` (that throws an exception if it's not overriden in the leaf class) would give us better error messages if someone forgets to define the verbose name.